### ISSUE TITLE
DateTime not using `defaultZoneName` set via `Settings`

### ISSUE THREAD

**Issue by mkascel** [moment/luxon#454]
I'm trying to write Node/Jest tests for an implementation and it looks like I am not able to override the default timezone as described in the docs [here](https://github.com/moment/luxon/blob/master/docs/zones.md#changing-the-default-zone).  The implementation itself (React) works fine but something is not sticking on the node side; DateTime is correctly reporting the new `zoneName` but is not formatting with that zone.  

This is what I'm seeing in the node shell (my timezone is Berlin/GMT+1):

```js
$ node
> const { Settings, DateTime } = require('luxon')
undefined
> DateTime.local().zoneName
'Europe/Berlin'
> DateTime.fromSeconds(1541590322, { zone: 'utc' }).setZone('local').toFormat('EEE, dd MMM y, T');
'Wed, 07 Nov 2018, 12:32'
> Settings.defaultZoneName = "America/Los_Angeles"
'America/Los_Angeles'
> DateTime.local().zoneName
'America/Los_Angeles'
> DateTime.fromSeconds(1541590322, { zone: 'utc' }).setZone('local').toFormat('EEE, dd MMM y, T');
'Wed, 07 Nov 2018, 12:32'
> DateTime.local().zoneName
'America/Los_Angeles'
```

My understanding is that this is fixed by https://github.com/moment/luxon/pull/423 (`Settings.defaultZone` is reporting the Los Angeles zone as expected also)

I've tried this with both node v8.11.4 and 10.15.1

Thank you in advance, I'm enjoying using this lib over Moment :)

**Comment by mkascel** [comment]
A couple of other things I forgot to note, in case this helps:

I am using the `full-icu` version of `node`, and manually setting both the timezone (instead of 'local') and using a locale works in node:

```node
> DateTime.fromSeconds(1541590322, { zone: 'utc' }).setZone('America/Los_Angeles').toFormat('EEE, dd MMM y, T');
'Wed, 07 Nov 2018, 03:32'
> DateTime.fromSeconds(1541590322, { zone: 'utc' }).setLocale('zh-Hans').setZone('America/Los_Angeles').toFormat('EEE, dd MMM y, T');
'周三, 07 11 2018, 03:32'
```

**Comment by icambron** [comment]
Yeah, this needs to return the default zone, not the local zone: https://github.com/moment/luxon/blob/master/src/impl/zoneUtil.js#L21

**Comment by icambron** [comment]
Fixed in #456

**Comment by mkascel** [comment]
That was fast, thanks! 👍
